### PR TITLE
feat: add to existing album

### PR DESCRIPTION
**Pull request by enguerran**
In the popup, you can create a new album as previously
or select an existing album like the following screenshot

### CODE REVIEW

**Comment by enguerran** [comment]
There is a lot to do in refactoring and testing. But I was asked to 🚢 the feature, so I :shipit: !

File: src/components/SelectAlbumItem.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        })
+      })
+  }
+
+  handleImageLoaded () {
+    this.setState({ isImageLoading: false })
+  }
+
+  render () {
+    console.log('SelectAbumItem', this.props)
```
Comment (kosssi) [inline src/components/SelectAlbumItem.jsx]: maybe can be delete?

File: src/components/SelectAlbumsForm.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+import { withError } from '../components/ErrorComponent'
+import { withEmpty } from '../components/Empty'
+
+import Loading from '../components/Loading'
+import SelectAlbumItem from '../components/SelectAlbumItem'
+
+import { fetchAlbums, createAlbumMangoIndex, getAlbumsList } from '../ducks/albums'
+
+const DumbAlbumsList = props => console.log(props) || (
```
Comment (kosssi) [inline src/components/SelectAlbumsForm.jsx]: a console.log here?

File: src/components/SelectAlbumItem.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      !isLoading &&
+        <div className={styles['pho-album']}>
+          <div
+            onClick={() => this.props.onSelectAlbum(album)}
+            className={styles['pho-album-link']}
+          >
+            <img
+              className={styles['pho-album-photo-item']}
+              onLoad={this.handleImageLoaded}
+              style={isImageLoading ? 'display:none' : ''}
```
Comment (kosssi) [inline src/components/SelectAlbumItem.jsx]: It's ugly, but we'll say nothing: p

File: src/styles/addToAlbum.styl [hunk 4]
```diff
@@ -7,3 +7,7 @@
     .coz-modal-section
         border-top 1px solid grey-09
         padding    1em 1.5em .5em !important
```
Comment (GoOz) [inline src/styles/addToAlbum.styl]: Oh fuck me. WTF is that `!important` doing here? 😠
I know it's not from you, at least not in this PR, but i needed to say this.

Comment (enguerran) [inline src/styles/addToAlbum.styl]: Yeah, we will need some rewriting for JS code and CSS code ;)

File: src/components/SelectAlbumItem.jsx (hunk 3 above)
Comment (enguerran) [inline src/components/SelectAlbumItem.jsx]: Why do you say that it's ugly?

**Comment by enguerran** [comment]
[(5) [PHO-19] ETU je peux ajouter des photos à un album existant depuis la vue "Photos"](https://trello.com/c/gbCEFcPa/245-5-pho-19-etu-je-peux-ajouter-des-photos-a-un-album-existant-depuis-la-vue-photos)

**Pull request merged**